.travis.yml

```diff
@@ -70,37 +70,79 @@ jobs:
         - if [[ "$git_status_out" == *"docs/README.md"* ]]; then echo "Error- Components version was not changed. Download pre-commit tool";exit 1; fi
 
     - stage: Roles tests
-      name: Search Role Verification
+      name: Search tests on Centos
+      env:
+        - "MOLECULE_ROLE_IMAGE=centos:7"
       script:
-        - molecule --version
-        - cd roles/search/
-        - molecule converge
-        - sleep 1m
-        - molecule verify
-        - molecule destroy
+        - cd roles/search && molecule test
 
     - stage: Roles tests
-      name: Common Role Verification
```

Comment:
Given all other roles are dependant on the common role and we doubled the number of tests I think we can remove testing this specific role

Reply:
in that way we are losing the tests already written for that particular role

Can make sense to avoid running suites on multiple platforms that are going to be tested in another suite, so I will leave only one run for common, java, tomcat roles (I would prefer ubuntu 20.04 that is newer than centos:7)

Reply:
make sense